core-concepts/wallets/veworld/user-guide/setup.md

```diff
@@ -0,0 +1,35 @@
+# Setup
+
+
+## Install the application <a href="#install" id="install"></a>
+
+{% tabs %}
+
+{% tab title="Mobile" %} 
+
+You can install the app on your device by following this links:
+
+- [VeWorld on Play Store](https://play.google.com/store/apps/details?id=org.vechain.veworld.app&utm_source=docs_vechain&utm_medium=website&utm_campaign=vechain_communication)
+- [VeWorld on Apple Store](https://apps.apple.com/us/app/veworld/id6446854569?campaign=docs_vechain)
+
+If you cannot access a marketplace, you can find the latest APK on [veworld.com](https://www.veworld.com/)
+
+{% endtab %}
+
+{% tab title="Extension" %}
+
+For all Chromium-based Browsers, you can visit the [Chrome Extension Marketplace](https://chromewebstore.google.com/detail/veworld/ffondjhiilhjpmfakjbejdgbemolaaho?utm_source=docs_vechain&utm_medium=website&utm_campaign=vechain_communication)
+
+{% endtab %}
+
+{% endtabs %}
+
+## Back up wallet <a href="#back-up-wallet" id="back-up-wallet"></a>
+
+Once the wallet is created, we recommend you back up the wallet immediately
+
+The mnemonic words store all the information needed at any point in time to recover your wallet. Please back up your wallet once the wallet is created.
```

Comment:
- I would go for "backup" instead of "back up", in the app we have the unique word AFAIK.
- IMHO the word "wallet" is repeated too many times. 
- Backup of the wallet at creation should be more than a recommendation.

Reply:
Done! :)